### PR TITLE
Tests: Past msg for committee

### PR DESCRIPTION
**Pull request by MatheusFranco99**
This PR adds the following tests for the committee structure:
- [x] Past message for finished duty
- [x] Past message for non-existing duty

### CODE REVIEW

File: ssv/spectest/tests/committee/past_msg_duty_finished.go [hunk 1]
```diff
@@ -1,11 +1,119 @@
...
+			Identifier: msgID,
+			Root:       root,
+		}
+		signed := testingutils.SignQBFTMsg(ks.OperatorKeys[1], 1, msg)
+		signed.FullData = fullData
+
+		return signed
+	}
+
+	expectedError := "failed processing consensus message: could not process msg: invalid signed message: proposal is not valid with current state"
```
Comment (GalRogozinski) [inline ssv/spectest/tests/committee/past_msg_duty_finished.go:80]: looking at the code I would think the error should be "wrong msg height"

File: ssv/spectest/tests/committee/past_msg_duty_does_not_exist.go [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+	"fmt"
+
+	"github.com/ssvlabs/ssv-spec/qbft"
+	"github.com/ssvlabs/ssv-spec/ssv/spectest/tests"
+	"github.com/ssvlabs/ssv-spec/types"
+	"github.com/ssvlabs/ssv-spec/types/testingutils"
+)
+
+// PastMessageDutyDoesNotExist tests a valid proposal past msg for a duty that doesn't exist
+func PastMessageDutyDoesNotExist() tests.SpecTest {
```
Comment (GalRogozinski) [inline ssv/spectest/tests/committee/past_msg_duty_does_not_exist.go:13]: good grammar change :-)

**Review by GalRogozinski — CHANGES_REQUESTED** [review]

**Review by GalRogozinski — COMMENTED** [review]
just explain the error...
if it is a bug that existed

File: ssv/spectest/tests/committee/past_msg_duty_finished.go (hunk 1 above)
Comment (MatheusFranco99) [reply ssv/spectest/tests/committee/past_msg_duty_finished.go:80]: But in this runner the height is correct. Remember that a message is redirected to the runner that is assigned to the message's slot.

**Review by GalRogozinski — APPROVED** [review]

**Pull request merged**